apps/signing/tests/test_views.py

```diff
@@ -141,6 +151,10 @@ def test_version_failed_review(self):
         assert response.status_code == 202
         assert 'processed' in response.data
 
+        response = self.get(self.url(self.guid, '3.0'))
```

Comment:
it's not totally obvious from the test name why this is here. Ideally, I like to see things such as this in a separate test function but at the least I think you could just add a comment to explain what it's testing for.